On macos 10.15:

```
builder for '/nix/store/d2n3qffxd5niadqwbkv1ab1l2908pa3d-ruby2.7.5-language_detector-0.1.2.drv' failed with exit code 1; last 10 log lines:
  hint: 
  hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
  hint: 'development'. The just-created branch can be renamed via this command:
  hint: 
  hint:         git branch -m <name>
  Initialized empty Git repository in /private/tmp/nix-build-ruby2.7.5-language_detector-0.1.2.drv-0/language_detector-8910279/.git/
  installing
  buildFlags:
  Traceback (most recent call last):
  /nix/store/xpccsnlq8w8xnhmqsi8b9r37s995zyff-nix-bundle-install.rb:121:in `<main>': undefined method `with_build_args' for #<Bundler::RubygemsIntegration:0x00007fcbac181288> (NoMethodError)
```

preventing build of `anystyle-cli`

--- a/pkgs/development/ruby-modules/bundler/default.nix
+++ b/pkgs/development/ruby-modules/bundler/default.nix
@@ -4,8 +4,8 @@ buildRubyGem rec {
   inherit ruby;
   name = "${gemName}-${version}";
   gemName = "bundler";
-  version = "2.2.24";
-  source.sha256 = "1x3czmqhlyb593ap7mxkk47idi2jnbnrpwj8xlsjdpi7iair9y62";
+  version = "2.3.6";
+  source.sha256 = "1531z805j3gls2x0pqp2bp1vv1rf5k7ynjl4qk72h8lpm1skqk9r";
   dontPatchShebangs = true;
 
   postFixup = ''

--- a/pkgs/development/ruby-modules/gem/nix-bundle-install.rb
+++ b/pkgs/development/ruby-modules/gem/nix-bundle-install.rb
@@ -118,9 +118,7 @@ else
   source = Bundler::Source::Path.new(options)
 end
 spec = source.specs.search_all(name).first
-Bundler.rubygems.with_build_args build_flags do
-  source.install(spec)
-end
+source.install(spec, :build_args => build_flags)
 
 msg = spec.post_install_message
 if msg

